code/lib/cli-storybook/src/bin/index.ts

```diff
@@ -40,6 +41,38 @@ const command = (name: string) =>
     .option('--debug', 'Get more logs in debug mode', false)
     .option('--enable-crash-reports', 'Enable sending crash reports to telemetry data');
 
+command('init')
+  .description('Initialize Storybook into your project')
+  .option('-f --force', 'Force add Storybook')
+  .option('-s --skip-install', 'Skip installing deps')
+  .option('--package-manager <npm|pnpm|yarn1|yarn2>', 'Force package manager for installing deps')
+  .option('--use-pnp', 'Enable PnP mode for Yarn 2+')
+  .option('-p --parser <babel | babylon | flow | ts | tsx>', 'jscodeshift parser')
+  .option('-t --type <type>', 'Add Storybook for a specific project type')
+  .option('-y --yes', 'Answer yes to all prompts')
+  .option('-b --builder <webpack5 | vite>', 'Builder library')
+  .option('-l --linkable', 'Prepare installation for link (contributor helper)')
+  .option(
+    '--dev',
+    'Launch the development server after completing initialization. Enabled by default (default: true)',
+    process.env.CI !== 'true' && process.env.IN_STORYBOOK_SANDBOX !== 'true'
+  )
+  .option(
+    '--no-dev',
+    'Complete the initialization of Storybook without launching the Storybook development server'
+  )
+  .action((options) =>
+    withTelemetry('init', { cliOptions: options }, async () => {
+      await initiate(options);
+      if (!options.disableTelemetry) {
+        await telemetry('init', { source: 'cli' });
+      }
+    }).catch((e) => {
+      logger.error(e);
+      process.exit(1);
+    })
+  );
+
```

Comment:
It would be great if you could re-use this code somehow:

https://github.com/storybookjs/storybook/blob/aec16d5b490d238050b6bdec3d169c77ed70498d/code/lib/create-storybook/src/bin/index.ts#L16-L50

Reply:
This code will only be used for the help command `storybook --help`, because otherwise it gets forwarded to the right file using this proxy:

https://github.com/storybookjs/storybook/blob/3416d197992e46fb05f8fcc16767253f2e34572e/code/lib/cli/src/proxy.ts#L1-L22

Reply:
> It would be great if you could re-use this code somehow:
> 
> https://github.com/storybookjs/storybook/blob/aec16d5b490d238050b6bdec3d169c77ed70498d/code/lib/create-storybook/src/bin/index.ts#L16-L50

I tried to reuse the code, but the applied options such as addToGlobalContext are different, so it will be difficult to reuse.

Reply:
That is correct.

Reply:
Thank you!

Reply:
> That is correct.

Thank you for review! I've applied the requirements that you mentioned! 🙂